docker-compose.yml

```diff
@@ -21,3 +22,20 @@ services:
       - ./ccm_web/dist:/base/dist:z
     command: npm run dev
     container_name: ccm_web
+    depends_on:
+      - database
+
+  database:
+    image: postgres:10.4-alpine
```

Comment:
What is the reason to chose 10.4 version when [latest](https://www.postgresql.org/about/news/postgresql-13-released-2077/) is 13?

Reply:
I think I copied this from ROHQ. I assumed that's what the MiDatabase team is offering. Is there somewhere we can check this?

Reply:
I can check with them

Reply:
So MiDatabase team offer Postgres in AWS (Cloud) through the ITS Container Service.https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_PostgreSQL.html#PostgreSQL.Concepts.General.DBVersions.
@ssciolla We can choose any version as long as it's available on AWS. Let's go with latest??

Reply:
Okay, I'll update it to a reasonable latest version.

Reply:
I updated it to 13 for now, but see #49.